### DISCUSSION TITLE
Private cloud sync

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [uBlockOrigin/uBlock-issues#2734] [Ideas]
Hi,

I really do not want to use Firefox sync and for sure not any Google cloud space.
It would be awesome if you can add a more private possibility to sync the data with a between devices.
I think webdav could be a good choice.

Thanks and regards
Hyper

**Comment by gothic-bum** [comment]
Previously suggested and declined:
https://github.com/gorhill/uBlock/issues/2468

**Comment by HyperCriSiS** [comment]
> Previously suggested and declined: [gorhill/uBlock#2468](https://github.com/gorhill/uBlock/issues/2468)

Seems it was declined without any reason :(
Really sad and makes no sense to me.

**Reply by stephenhawk8054** [reply]
The list of demands is too many. Dev has limited time for the features that do not affect blocking abilities. Currently there are many issues about blocking functionalities that are still waiting to be fixed. Dev has to prioritize the issues.

People usually demand things without knowing how complicated that would cost. Even blocking functions need months/years to build a concept, develop, fine-tune and mature.

Also, if a feature requires sending information, it would need extreme consideration.

**Comment by gorhill** [comment]
> without any reason
> makes no sense to me

The reason is that I decline to work on things I don't want to work on, I choose to not invest my personal time on what I consider a lot of work to implement/maintain, but really what make no sense is for you to require that other people need to justify to you why they don't want to work on your ideas.
